lib/validations.js

```diff
@@ -776,7 +776,7 @@ function ErrorCodes(messages) {
  *   }
  *
  *   // This line shows how to create a ValidationError
- *   err = new ValidationError(model);
+ *   var err = new MyModel.ValidationError(model);
```

Comment:
Using it like that in a remote method throws a `ValidationError is not defined` error.
Using proposed method works, the ideas is from here: https://github.com/strongloop/loopback/blob/master/lib/persisted-model.js#L438